contracts/DutchExchange.sol

```diff
@@ -1070,6 +1074,38 @@ contract DutchExchange is DxUpgrade, TokenWhitelist, EthOracle {
         auctionIndex = latestAuctionIndices[token1][token2];
     }
 
+    function setClearingTime(
+        address token1,
+        address token2,
+        uint auctionIndex,
+        uint auctionStart,
+        uint sellVolume,
+        uint buyVolume
+    )
+        internal
+    {
+        (uint pastNum, uint pastDen) = getPriceInPastAuction(sellToken, buyToken, auctionIndex - 1);
+        // timeElapsed = 43200(2 * pastNum * sellVolume - buyVolume * pastDen)/(sellVolume * pastNum + buyVolume * pastDen)
+        uint numerator = sub(mul(mul(pastNum, sellVolume), 86400), mul(mul(buyVolume, pastDen), 43200));
```

Comment:
Can we use more semantic time units? https://solidity.readthedocs.io/en/v0.4.21/units-and-global-variables.html#time-units

Reply:
This is imho better because it is consistent with the numbers used elsewhere (lines 782, 963, 965). I will add comment explaining that it's 12 and 24 hrs respectively

Reply:
In my opinion we should change it in the DutchX too, I think we should always use semantic times